anta/cli/check/commands.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+# pylint: disable=no-value-for-parameter
+# pylint: disable=too-many-arguments
+
+"""
+Commands for Anta CLI to run check commands.
+"""
+
+import re
+import logging
+import click
+
+from rich.console import Console
+from rich.theme import Theme
+from rich.panel import Panel
+
+from anta import RICH_COLOR_THEME
+from anta.cli.cli import anta
+from .utils import check_run, display_table, display_json, display_list
+
+logger = logging.getLogger(__name__)
+
+
+@anta.command(no_args_is_help=True)
+@click.pass_context
+# Generic options
+@click.option('--inventory', '-i', show_envvar=True, prompt='Inventory path', help='Path to your inventory file', type=click.Path())
+@click.option('--catalog', '-c', show_envvar=True, prompt='Path for tests catalog', help='Path for tests catalog', type=click.Path())
+@click.option('--tags', '-t', default='all', help='List of tags using coma as separator: tag1,tag2,tag3', type=str)
+@click.option('--display', default='table', type=click.Choice(['table', 'json', 'list'], case_sensitive=False), help='output format selection. default is table')
```

Comment:
Can we find a way via click to avoid repeating the options for both command (like some kind of "group of options" that could be apploed to both)?

Reply:
- Have not seen such option in Click or its plugins.
- So I have moved inventory under cli like username and password.
- Cagtalog and display are not for all entrypoint so it should not be changed.
- Tags is still present at this level as it is probably the most updated element to run CLI